cdap-source-control/src/main/java/io/cdap/cdap/sourcecontrol/RepositoryManager.java

```diff
@@ -65,11 +75,13 @@ public class RepositoryManager implements AutoCloseable {
   // same namespace from interfering with each other.
   private final String randomDirectoryName;
 
-  public RepositoryManager(SecureStore secureStore, SourceControlConfig sourceControlConfig) {
-    this.sourceControlConfig = sourceControlConfig;
+  @Inject
```

Comment:
see the other comment, don't use guice to create this. Instead, have actual classes that implement the factory and create these instances directly.

Reply:
Removed guice inject, instead now we're using a concrete `RepositoryManagerFactory`